Software/dist_windows/script.iss

```diff
@@ -67,11 +64,9 @@ Source: "content/prismatik-unhook.dll"; DestDir: "{app}"; MinVersion: 6.1.7600;
 Source: "content/prismatik-hooks32.dll"; DestDir: "{app}"; MinVersion: 6.1.7600; Flags: 32bit ignoreversion
 Source: "content/prismatik-unhook32.dll"; DestDir: "{app}"; MinVersion: 6.1.7600; Flags: 32bit ignoreversion
 Source: "content/platforms/*"; DestDir: "{app}\platforms"; Flags: ignoreversion
-Source: "content/translations/*.qm"; DestDir: "{app}\translations"; Flags: ignoreversion
 Source: "content/Plugins/*"; DestDir: "{#UserSettingsDirName}\Plugins"; Flags: onlyifdoesntexist createallsubdirs recursesubdirs
-; These files are needed only when building with MSVC - otherwise just comment them
-Source: "content/msvcr120.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "content/msvcp120.dll"; DestDir: "{app}"; Flags: ignoreversion
+; Visual Studio redistributable package
+Source: "content/vcredist_x64.exe"; DestDir: {tmp}; Flags: deleteafterinstall

```

Comment:
I see, I did not know that. Windows has a tendency for having multiple ways to achieve something, and everyone recommends something else. The Qt team thought this was the best idea, but it's always possible to deviate from that. I'll edit it back. Do you prefer the previous one where you copied it from the Windows directory, or the one that you can find in `<Visual Studio install dir>\VC\redist\<arch>\Microsoft.VC120.CRT`?